fips-tests/pom.xml

```diff
@@ -0,0 +1,86 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <parent>
+        <groupId>io.confluent</groupId>
+        <artifactId>rest-utils-parent</artifactId>
+        <version>7.7.0-0</version>
+    </parent>
+
+    <artifactId>rest-utils-fips-tests</artifactId>
+    <packaging>jar</packaging>
+    <name>rest-utils-fips-tests</name>
+
+    <dependencies>
```

Comment:
are the dependencies minimal? (i.e no redundant dep)

Reply:
AFAIK, they are all required. I will do one more iteration and raise revision.

Reply:
@trnguyencflt removed unnecessary deps.